src/components/resume-items/education-item.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Textarea } from "@/components/ui/textarea";
+import { useState } from "react"; // Import useState
+
+export function EducationItem() {
+  const [universityName, setUniversityName] = useState("");
+  const [startDate, setStartDate] = useState("");
+  const [endDate, setEndDate] = useState("");
+  const [major, setMajor] = useState("");
+  const [minor, setMinor] = useState("");
+  const [description, setDescription] = useState("");
+
+  const handleFormSubmit = async (event: any) => {
+    event.preventDefault();
+
+    const educationData = {
+      universityName,
+      startDate,
+      endDate,
+      major,
+      minor,
+      description,
+    };
+
+    console.log(educationData);
+
+    // API call to save data (replace placeholder with your actual implementation)
+    try {
+      const response = await fetch("/api/save-education-data", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(educationData),
+      });
+
+      if (response.ok) {
+        // Handle successful submission (e.g., close dialog, clear inputs, display feedback)
+      } else {
+        // Handle error from API
+      }
+    } catch (error) {
+      console.error("Error submitting form:", error);
+    }
+  };
+
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button className="text-left h-full w-full" variant="ghost">
+          Education
+        </Button>
+      </DialogTrigger>
+      <DialogContent className="sm:max-w-[600px]">
+        <DialogHeader>
+          <DialogTitle>Add Education</DialogTitle>
+          <DialogDescription>
+            Fill in the following information
+          </DialogDescription>
+        </DialogHeader>
+        <form onSubmit={handleFormSubmit}>
+          <div className="grid grid-cols-2 gap-4 flex">
+            <Input
+              className="w-[300px]"
+              id="item-name"
+              placeholder="University Name"
+              value={universityName}
+              onChange={(e) => setUniversityName(e.target.value)}
+            />
+            <div className="flex justify-end">
+              <Input
```

Comment:
I was thinking it might make more sense to make date one field, since they might sometimes want to do something like "May 2023 - July 2023" or might just want to do "Jan 2023" so it might be better to leave the control completely up to them. This would apply for all the forms. Lmk what you think

Reply:
Yeah I initially didn't consider that. Good call, we'll have to find a way to ensure correct formatting, but I'll look into that later.